chi/context.py

```diff
@@ -237,8 +252,44 @@ def params():
     keys.extend(list(cfg.CONF[_auth_section(_auth_plugin)].keys()))
     return keys
 
+def list_sites(show: Optional[str] = None) -> List[str]:
+    """
+    Retrieve a list of Chameleon sites.
+
+    Args:
+        show (str, optional): Determines how the list of sites is displayed.
+            Possible values are "widget" and "text". Defaults to None.
+
+    Returns:
+        List[str]: A list of site names.
+
+    Raises:
+        ValueError: If no sites are returned.
+
+    """
+    global _sites
+    if not _sites:
+        res = requests.get(f"{RESOURCE_API_URL}/sites.json")
+        try:
+            res.raise_for_status()
+            items = res.json().get("items", [])
```

Comment:
We should manually insert KVM, see https://github.com/ChameleonCloud/python-chi/blob/master/chi/context.py#L297-L303

This is a bit hacky, since we are re-using the hardware API for this, and KVM is the only site outside of the hardware API.